.gitignore

```diff
@@ -1,4 +1,5 @@
 *.iml
+generated/
```

Comment:
API client samples should be put under `./samples/client/petstore` as an example.